### PR TITLE
GC 2023: Daniel Gomez Blanco self-nomination

### PR DESCRIPTION
**Pull request by danielgblanco**

### CODE REVIEW

**Comment by mhausenblas** [comment]
Very happy to see this, and I support Dan's nomination.

**Review by codeboten — APPROVED** [review]
🎉

File: elections/2023/governance-committee-candidates.md [hunk 1]
```diff
@@ -35,6 +35,48 @@ will prioritize efforts to increase the contributor and maintainer community
 size, ensure that the project is welcoming and inclusive, and to guide the
 project towards graduation and long-term success.
 
+### Daniel Gomez Blanco
```
[the remark this answers is not in the document]
Comment (trask) [reply elections/2023/governance-committee-candidates.md:38]: it looks like we've sorted by first name previously https://github.com/open-telemetry/community/blob/main/elections/2022/governance-committee-candidates.md

**Review by trask — APPROVED** [review]

File: elections/2023/governance-committee-candidates.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (danielgblanco) [reply elections/2023/governance-committee-candidates.md:38]: Cool, so then I'll keep it this way. I can move Eric up in the list in this PR if it's not fixed elsewhere.

[the remark this answers is not in the document]
Comment (trask) [reply elections/2023/governance-committee-candidates.md:38]: thx, due to merge conflicts, probably best for us to send a PR to re-alphabetize after nominations close (cc @dyladan)

**Comment by svrnm** [comment]
I support!

**Review by arminru — APPROVED** [review]

**Pull request merged**